### PR TITLE
Create technology-program

### PR DESCRIPTION
**Pull request by traci707**
Partner Solutions Project

# [DEVDOCS-2243](https://jira.bigcommerce.com/browse/DEVDOCS-2243)

## What changed?
Updated the structure for tech partners

Here is the outline I followed:
Partner Program /single article/
Partner Program Overview
Program Application / Requirements
Partner Portal Overview
App Marketplace Overview

### CODE REVIEW

**Review by uwikaiddi — APPROVED** [review]

File: docs/api-docs/partner/technology-program [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# Technology Partner Program
```
Comment (ektravel) [inline docs/api-docs/partner/technology-program:1]:
```suggestion
# Partner Program
```

File: docs/api-docs/partner/technology-program [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+<div class="otp" id="no-index">
+
+### On this page
+- [Technology Partner Program overview](#program-overview)
```
Comment (ektravel) [inline docs/api-docs/partner/technology-program:6]:
```suggestion
- [Program overview](#program-overview)
```

File: docs/api-docs/partner/technology-program [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+- [Technology Partner Program application/requirements](#partner-app-requirements)
```
Comment (ektravel) [inline docs/api-docs/partner/technology-program:7]:
```suggestion
- [Application requirements](#application-requirements)
```

File: docs/api-docs/partner/technology-program [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+- [Technology Partner Program portal overview](#program-portal-overview)
```
Comment (ektravel) [inline docs/api-docs/partner/technology-program:8]:
```suggestion
- [Partner portal](#partner-portal)
```

File: docs/api-docs/partner/technology-program [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+- [App Marketplace overview](#app-marketplace-overview)
```
Comment (ektravel) [inline docs/api-docs/partner/technology-program:9]:
```suggestion
- [App Marketplace](#app-marketplace)
```

File: docs/api-docs/partner/technology-program [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+- [Troubleshooting](#troubleshooting)
+- [Resources](#resources)
+</div> 
+
+## Technology Partner Program overview
```
Comment (ektravel) [inline docs/api-docs/partner/technology-program:14]:
```suggestion
## Program overview
```

File: docs/api-docs/partner/technology-program [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+Join the Technology Partner Program and become an official partner. As an official partner, you will have access to training, sales enablement, and a referral network to get more clients and build better ecommerce experiences.
```
Comment (ektravel) [inline docs/api-docs/partner/technology-program:15]:
```suggestion
Join the BigCommerce Partner Program to become an official BigCommerce partner. As an official partner, you will have access to training, sales enablement, and a referral network to get more clients and build better ecommerce experiences.
```

File: docs/api-docs/partner/technology-program [hunk 8]
```diff
@@ -0,0 +1,132 @@
...
+
+See the Technology Partner Program application/requirements in the section below to find out how to apply.
```
Comment (ektravel) [inline docs/api-docs/partner/technology-program:17]:
```suggestion
For instruction on how to apply, review the Partner Program [application requirements](#application-requirements).
```

File: docs/api-docs/partner/technology-program [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+
+## Technology Partner Program application/requirements
```
Comment (ektravel) [inline docs/api-docs/partner/technology-program:19]:
```suggestion
## Application requirements
```

File: docs/api-docs/partner/technology-program [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+Only official partners can submit an app to the BigCommerce App Marketplace, also known as the App store. BigCommerce offers app developers free sandbox stores through its Technology Partner Program. To be approved as a partner, you will need:
```
Comment (ektravel) [inline docs/api-docs/partner/technology-program:20]:
```suggestion
Only official partners can submit an app to the BigCommerce App Marketplace, also known as the App store. BigCommerce offers app developers free sandbox stores through its Partner Program. To be approved as a partner, you will need the following:
```

File: docs/api-docs/partner/technology-program [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+
+* A website showcasing your agency or technology solution
+* Intent to offer your app publicly
+* Ability to support your app's users
+* Evidence of expertise in app development
+
+Also, there is a list of general requirements in [App Store Approval Requirements](https://developer.bigcommerce.com/api-docs/partner/app-store-approval-requirements), which provides best practices for you to follow.
```
Comment (ektravel) [inline docs/api-docs/partner/technology-program:27]:
```suggestion
For a list of general requirements and to verify that your application meets BigCommerce listing standards, see [App Store Approval Requirements](https://developer.bigcommerce.com/api-docs/partner/app-store-approval-requirements).
```

File: docs/api-docs/partner/technology-program [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+4. Click the **Technology Partner Program** button.
+   
+![Partner Registration](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/becoming-a-partner-02.png "Partner Registration")
+
+5. Complete the member registration.
+6. Click the **Complete Registration** button. 
+7. Once your application is approved, check your email for your Partner ID.
+
+
+Once approved, you will receive one or more emails listing your partner credentials and next steps. Your **Partner ID** will be required to submit your app for App Marketplace consideration. If you do not receive these credentials by email, please contact [appstore@bigcommerce.com](mailto:appstore@bigcommerce.com).
```
Comment (ektravel) [inline docs/api-docs/partner/technology-program:47]:
```suggestion
Once approved, you will receive one or more emails listing your partner credentials and next steps. Your Partner ID will be required to submit your app for App Marketplace consideration. If you do not receive these credentials by email, please contact [appstore@bigcommerce.com](mailto:appstore@bigcommerce.com).
```

File: docs/api-docs/partner/technology-program [hunk 13]
```diff
@@ -0,0 +1,132 @@
...
+
+## App Marketplace Overview
```
Comment (ektravel) [inline docs/api-docs/partner/technology-program:49]:
```suggestion
## App Marketplace
```

File: docs/api-docs/partner/technology-program [hunk 14]
```diff
@@ -0,0 +1,132 @@
...
+
+### Trials erased after 15 days
+> Although you do not need a sandbox store for app development, it is best practice. If you opt to develop on a trial store, keep in mind the store and data will be erased after 15 days, and you will have to create a new trial store to proceed.
+
+
+</div>
+</div>
+</div>
+
+### Creating a sandbox store
```
Comment (ektravel) [inline docs/api-docs/partner/technology-program:75]: I think this section should be removed from this article and included in the BigCommerce API Basics -> Getting started.
```suggestion
### Creating a sandbox store
```

File: docs/api-docs/partner/technology-program [hunk 15]
```diff
@@ -0,0 +1,132 @@
...
+
+5. Navigate to **Support > Contact Support** and select **Create a Support Case**.
+
+6. While completing the form, use "Partner Sandbox" as the subject line. In the case description, enter your new sandbox store's canonical URL (for example: http://your-store-name.mybigcommerce.com) and request conversion to a sandbox. 
+
+7. Under **Issue Type**, select **Merchant Account Services**. Click **Create Case**. We will flag this new trial store as a sandbox to ensure your store access continues beyond the 15-day trial period. This process can take several days.
+
+8.  Log into your new trial store using the same email associated with your partner login.
+
+### Marketplace apps
```
Comment (ektravel) [inline docs/api-docs/partner/technology-program:93]: I think this section should be removed from this article and included under Application Development.
``suggestion
### Marketplace apps
```

File: docs/api-docs/partner/technology-program [hunk 16]
```diff
@@ -0,0 +1,132 @@
...
+* Within the [Dev Tools](https://developer.bigcommerce.com/api-docs/getting-started/authentication/rest-api-authentication#obtaining-app-api-credentials) workspace, you must enable your app's **Technical** > **Multiple Users** option.
+
+In the payload returned when a user launches an app, users are distinguished by `owner_email` versus `user_email`. If these two emails match, the user is the store owner.
+
+If you wish to enable user removal, you can do by filling in your app's **Technical** > **Remove User Callback URL** field in Dev Tools. Enabling user removal is optional. For more advanced implementations, you can enable the store owner to grant specific permissions to different non-admin users. For example, `person1@email.com` could be restricted to only editing product inventory but not viewing orders. If you decide to include user permission in your app, it’s a great feature to advertise.
+
+For more information, see [Multi-User Support](/api-docs/apps/guide/intro#multi-user-support).
+
+## Troubleshooting
+##  Troubleshooting
```
Comment (ektravel) [inline docs/api-docs/partner/technology-program:106]: Same as line 105
```suggestion
```

File: docs/api-docs/partner/technology-program [hunk 17]
```diff
@@ -0,0 +1,132 @@
...
+* Tokens must be stored against the `store_hash` and not against user info.
+* Within the [Dev Tools](https://developer.bigcommerce.com/api-docs/getting-started/authentication/rest-api-authentication#obtaining-app-api-credentials) workspace, you must enable your app's **Technical** > **Multiple Users** option.
+
+In the payload returned when a user launches an app, users are distinguished by `owner_email` versus `user_email`. If these two emails match, the user is the store owner.
+
+If you wish to enable user removal, you can do by filling in your app's **Technical** > **Remove User Callback URL** field in Dev Tools. Enabling user removal is optional. For more advanced implementations, you can enable the store owner to grant specific permissions to different non-admin users. For example, `person1@email.com` could be restricted to only editing product inventory but not viewing orders. If you decide to include user permission in your app, it’s a great feature to advertise.
+
+For more information, see [Multi-User Support](/api-docs/apps/guide/intro#multi-user-support).
+
+## Troubleshooting
```
Comment (ektravel) [inline docs/api-docs/partner/technology-program:105]: I think Troubleshooting should be removed from the Overview article and included in Application Development -> Development Guidelines.

File: docs/api-docs/partner/technology-program [hunk 18]
```diff
@@ -0,0 +1,132 @@
...
+
+**Who should I contact if I have further questions?**
+
+If you have additional questions, email <a href="mailto:appstore@bigcommerce.com">appstore@bigcommerce.com</a>.
+
+## Resources
+
+### Related articles
+* [App Store Approval Requirements](https://developer.bigcommerce.com/api-docs/partner/app-store-approval-requirements)
+* [Partner Portal](https://partners.bigcommerce.com/) (BigCommerce)
```
Comment (ektravel) [inline docs/api-docs/partner/technology-program:131]:
```suggestion
```

**Review by ektravel — COMMENTED** [review]
Added my suggestions.

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**